### DISCUSSION TITLE
音量控制没反应

### DISCUSSION THREAD

**Discussion by koolyang** [MonitorControl/MonitorControl#1278] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

我的外接显示屏是led cinema display

### Steps to reproduce

点击控制面板滑动音量条

### Expected behavior

希望可以控制外接屏音量

### Environment Information (please complete the following information)

```markdown
- macOS version:10.15.7 (19H1715)
- Mac model:19 macbookpro
- MonitorControl version:v4.0.2
- Monitor(s):apple LED cinema display
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
I am afraid the Apple LED Cinema Display does not allow volume control via DDC. It should have a built-in USB control though but that support might have been removed from recent macOS versions.

**Comment by koolyang** [comment]
这个版本使用时会把自带的系统音量控制按钮禁用，所以现在使用起来只能通过其他软件自带的音量控制器调整音量；但在很久之前我用的一个版本是可以控制音量的，更新后就不行了。

**Comment by waydabber** [comment]
Translation: _When this version is used, the built-in system volume control button will be disabled, so now you can only adjust the volume through the volume controller that comes with other software; but a version I used a long time ago can control the volume, after the update Not anymore._

--

So the question is what do you mean by a version used long time ago. Apple LED Cinema Displays had native support back in the day in macOS but this was removed. I think this is what causes the problem.

**Comment by jj-25** [comment]
same issue.
device: macbook m1 pro 2021
macOS: 13.0.1
Monitor: MSI MAG274QRF

**Comment by R33GTR-TobyWang-1981** [comment]
同样的问题
